lib/spack/spack/cmd/install.py

```diff
@@ -41,6 +41,8 @@
 from spack.package import PackageBase
 
 description = "build and install packages"
+section = "basic"
+level = "short"
```

Comment:
In the context of `spack help --all`, it makes more sense for `install` and `uninstall` to go in Building.